core/database/migrations/20160723182900_createSchema.sql

```diff
@@ -0,0 +1,210 @@
+/* Create Schema
+ *
+ */
+
+BEGIN;
+
+/* Tables */
+CREATE TABLE IF NOT EXISTS "migrations" (
+  "id"   VARCHAR(25) NOT NULL PRIMARY KEY,
+  "name" TEXT        NOT NULL
+);
+
+CREATE TABLE IF NOT EXISTS "blocks" (
+  "id"                   CHAR(64) PRIMARY KEY,
+  "rowId"                SERIAL NOT NULL,
+  "version"              INT    NOT NULL,
+  "timestamp"            INT    NOT NULL,
+  "height"               INT    NOT NULL,
+  "previousBlock"        CHAR(64),
+  "numberOfTransactions" INT    NOT NULL,
+  "totalAmount"          BIGINT NOT NULL,
+  "totalFee"             BIGINT NOT NULL,
+  "reward"               BIGINT NOT NULL,
+  "payloadLength"        INT    NOT NULL,
+  "payloadHash"          BYTEA  NOT NULL,
+  "generatorPublicKey"   BYTEA  NOT NULL,
```

Comment:
Do we need to store all such fields in `hex` format?

Reply:
I don't know